en/frames_multicopter/holybro_x500V2_pixhawk5x.md

```diff
@@ -0,0 +1,211 @@
+# Holybro X500 V2 + Pixhawk5x Build
+
+This topic provides full instructions for building the kit and configuring PX4 using *QGroundControl*.
+
+## Key information
+
+- **Kit:** [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html)
+- **Frame:** [Holybro X500 V2 Frame Kit](https://shop.holybro.com/x500-v2-kit_p1288.html)
+- **Flight controller:** [Pixhawk 5X](../flight_controller/pixhawk5x.md)
+- **Assembly time (approx.):** 55 min (25 minutes for frame, 30 minutes for autopilot installation/configuration)
+
+![Full X500 V2 Kit](../../assets/airframes/multicopter/x500_V2_holybro_pixhawk5x/X500-kit.png)
+
+## Bill of materials
+
+The Holybro [X500 V2 Kit](http://shop.holybro.com/x500-v2-kit_p1288.html) includes almost all the required components:
+
+* X500V2 Frame Kit
+  * Body - Full Carbon Fiber Top & Bottom plate (144 x 144mm, 2mm thick)
+  * Arm - High strength & ultra-lightweight 16mm carbon fiber tubes
+  * Landing gear - 16mm & 10mm diameter carbon fiber tubes
+  * Platform board - With mounting holes for GPS & popular companion computer
+  * Dual 10mm Ø rod x 250 mm long rail mounting system
+  * Battery mount with two Battery Straps
+  * Hand tools for installation
+* [Holybro Motors - 2216 KV880 x6](https://shop.holybro.com/motor2216-880kv-1pc_p1154.html)
+* [Holybro BLHeli S ESC 20A x4](https://shop.holybro.com/blheli-s-esc-20a_p1143.html)
+* [Propellers - 1045 x4](https://shop.holybro.com/propeller10452pair_p1155.html)
+* Power Distribution Board – XT60 plug for battery & XT30 plug for ESCs & peripherals
+* Camera mount (optional and the 3D file can be downloaded from [here](http://www.holybro.com/3D_Print/Holybro_X500_V2_3D%20Print.rar))
+
+Other parts in this build:
```

Comment:
Are the GPS, Telemetry, and PM included in the Kit? if not instead of "Other Parts in this build" I recommend we change to "Not included in this kit (but covered in the guide)"

Reply:
100% agree with this. Essentially we want to identify the other things they need to buy.